openvpn/check.go

```diff
@@ -0,0 +1,45 @@
+package openvpn
+
+import (
+	"errors"
+	"os/exec"
+	"strconv"
+	"syscall"
+)
+
+//CheckOpenvpnBinary function checks that openvpn is available, given path to openvpn binary
+func CheckOpenvpnBinary(openvpnBinary string) error {
```

Comment:
Shouldn't we use abbrevation naming conventions for `vpn`, i.e. `CheckOpenVPNBinary`?

Reply:
Well its not an abrevation but more well known trademark "openvpn".